src/Protocol/ActivityPub/Receiver.php

```diff
@@ -437,17 +437,18 @@ public static function prepareObjectData(array $activity, int $uid, bool $push,
 		$object_data['receiver'] = array_replace($object_data['receiver'] ?? [], $receivers);
 		$object_data['reception_type'] = array_replace($object_data['reception_type'] ?? [], $reception_types);
 
-		$author = $object_data['author'] ?? $actor;
-		if (!empty($author) && !empty($object_data['id'])) {
-			$author_host = parse_url($author, PHP_URL_HOST);
-			$id_host = parse_url($object_data['id'], PHP_URL_HOST);
-			if ($author_host == $id_host) {
-				Logger::info('Valid hosts', ['type' => $type, 'host' => $id_host]);
-			} else {
-				Logger::notice('Differing hosts on author and id', ['type' => $type, 'author' => $author_host, 'id' => $id_host]);
-				$trust_source = false;
-			}
-		}
+//		This check here interferes with Hubzilla posts where the author host differs from the host the post was created
```

Comment:
Will this be dead code or something `@todo` in the future?

Reply:
A `@todo` is used as a reminder for further extensions of the code in the future. It isn't used for cases like this one where we deactivate some code.